deployment-template/ec2/data-prepper-ec2-deployment-cfn.yaml

```diff
@@ -70,43 +109,84 @@ Resources:
               owner: root
               group: root
             "/etc/data-prepper/pipelines.yaml":
-              content: !Sub |
-                entry-pipeline:
-                  delay: "100"
-                  source:
-                    otel_trace_source:
-                      ssl: false
-                      health_check_service: true
-                  sink:
-                    - pipeline:
-                        name: "raw-pipeline"
-                    - pipeline:
-                        name: "service-map-pipeline"
-                raw-pipeline:
-                  source:
-                    pipeline:
-                      name: "entry-pipeline"
-                  prepper:
-                    - otel_trace_raw_prepper:
-                  sink:
-                    - elasticsearch:
-                        hosts: [ "${AWSElasticsearchEndpoint}" ]
-                        aws_sigv4: true
-                        aws_region: "${AWSElasticsearchRegion}"
-                        trace_analytics_raw: true
-                service-map-pipeline:
-                  delay: "100"
-                  source:
-                    pipeline:
-                      name: "entry-pipeline"
-                  prepper:
-                    - service_map_stateful:
-                  sink:
-                    - elasticsearch:
-                        hosts: [ "${AWSElasticsearchEndpoint}" ]
-                        aws_sigv4: true
-                        aws_region: "${AWSElasticsearchRegion}"
-                        trace_analytics_service_map: true
+              content: !If
+                - NoMasterUser
+                - !Sub |
```

Comment:
Could this if-block be reduced to just the ES sink level? (there would be 2 if blocks but they would be smaller in scope)

Reply:
Taking a look